modules/grandpa/src/lib.rs

```diff
@@ -441,15 +452,38 @@ pub mod pallet {
 
 		let initial_hash = header.hash();
 		<InitialHash<T, I>>::put(initial_hash);
-		<BestFinalized<T, I>>::put(initial_hash);
-		<ImportedHeaders<T, I>>::insert(initial_hash, header);
+		<ImportedHashesPointer<T, I>>::put(0);
+		insert_header::<T, I>(header, Some(initial_hash));
 
 		let authority_set = bp_header_chain::AuthoritySet::new(authority_list, set_id);
 		<CurrentAuthoritySet<T, I>>::put(authority_set);
 
 		<IsHalted<T, I>>::put(is_halted);
 	}
 
+	#[cfg(feature = "runtime-benchmarks")]
+	pub(crate) fn bootstrap_bridge<T: Config<I>, I: 'static>(
+		init_params: super::InitializationData<BridgedHeader<T, I>>,
+	) {
+		let inject_headers = T::HeadersToKeep::get();
+		let start_number = *init_params.header.number();
+		let end_number = start_number + inject_headers.into();
```

Comment:
```suggestion
		let end_number = start_number + T::HeadersToKeep::get().into();
```

I think this is more readable than `inject_headers`